src/Dropdown/index.tsx

```diff
@@ -41,133 +44,161 @@ const X = styled('button')`
+const HiiMultiSelect: IDropdown = ({
+  onChange,
+  value = [],
+  theme,
+  ...props
+}) => {
+  // from here ->>
+  // could have been avoided if not tweaking react-select
+  // theming might be a nightmare due to uniqness of projects - from UI perpective
+  const HiiLabel = (labelProps: any) => {
+    return (
+      <>
+        <Placeholder {...labelProps} isFocused={labelProps.isFocused}>
+          {labelProps.selectProps.placeholder}
+        </Placeholder>
+        {React.Children.map(labelProps.children, (child: any) =>
+          child && child.type !== Placeholder ? child : null
+        )}
+      </>
+    )
+  }
+  const styles = {
+    multiValue: () => ({
+      display: 'none',
+    }),
+    option: (provided: any) => ({
+      ...provided,
+      ':hover': {
+        ...provided[':hover'],
+        backgroundColor: '#B6EFA0',
+        color: '#27847A',
+      },
+    }),
+    placeholder: (provided: any) => ({
+      ...provided,
+      position: 'absolute',
+      padding: '8px 4px',
+    }),
+    menuList: (provided: any) => ({
+      ...provided,
+      color: '#fff',
+      backgroundColor: '#27847A',
+    }),
+  }
+  // <<- to here
   const handleRemoveValue = (e: any) => {
     if (!onChange) return null
-    const { name: buttonName } = e.currentTarget
-    const removedValue = value.find((val: any) => val.value === buttonName)
+    const { name } = e.currentTarget
+    const removedValue = value.find((val: any) => val.value === name)
 
     if (!removedValue) return null
     props.update(value)
 
     onChange(
-      value.filter((val: any) => val.value !== buttonName),
+      value.filter((val: any) => val.value !== name),
       { name, action: 'remove-value', removedValue }
     )
   }
 
   return (
     <Wrapper>
       <Select
-        isMulti
-        theme={(theme) => ({
-          ...theme,
+        styles={styles}
+        theme={(provided) => ({
+          ...provided,
           borderRadius: 0,
         })}
-        {...props}
         components={{
-          ValueContainer: CustomLabel,
+          ValueContainer: HiiLabel,
         }}
         closeMenuOnSelect={true}
         controlShouldRenderValue={true}
+        {...props}
+        onChange={onChange}
       />
       <ValuesContainer>
         {value.map((val: any) => (
           <Value {...val} key={val.value}>
-            {val.label}
-            <X {...val} name={val.value} onClick={handleRemoveValue}>
-              ✕
-            </X>
+            <span>{val.label}</span>
+            <Button_X
+              color={val.textColor}
+              name={val.value}
+              onClick={handleRemoveValue}
+            >
+              x
+            </Button_X>
           </Value>
         ))}
       </ValuesContainer>
     </Wrapper>
   )
 }
 
-const Dropdown: IDropdown = ({ options, isMulti = false, ...props }) => {
+const Dropdown: IDropdown = ({
+  theme = 'default',
```

Comment:
This is a pretty ugly solution. Are these just not separate components? I'm not sure how/if this can be managed with a theming context so I'm not sure where you're going